### ISSUE TITLE
Where does "DataParallelWithCallback" in GaitSet/model/model.py come from?

### ISSUE THREAD

**Issue by Lebron8997** [AbnerHqC/GaitSet#4]
The following line seems strange in 'model.py'?
from sync_batchnorm import DataParallelWithCallback
Should there exist a file named 'sync_batchnorm.py‘ in 'model' directory?

**Comment by AbnerHqC** [comment]
It's a [package](https://github.com/vacancy/Synchronized-BatchNorm-PyTorch) used for parallel BN. As there is no BN layer in our model it can be replaced by torch.nn.DataParallel(). Thank you for pointing this out. I'll replace it with nn.DataParallel.
